cmd/main.go

```diff
@@ -15,12 +15,25 @@ import (
 	"github.com/benbjohnson/clock"
 	"github.com/gin-contrib/cors"
 	"github.com/gin-gonic/gin"
+
+	u "kg/procurement/cmd/utils"
 )
 
 // main is the entrypoint for the entire service
 func main() {
+	logger := u.NewConsoleLogger()
+
+	logger.Info("Starting...")
+	logger.Debug("Printing stuff")
+	logger.Error("Error occured")
+	// logger.Panic("Panic occured")
+	// logger.Fatal("Fatal occured")
+	// u.GeneralLogger.Println("Starting...")
+
+	u.GeneralLogger.Println("Loading configurations")
 	cfg := config.Load()
 
+	u.GeneralLogger.Println("Creating database connection")
```

Comment:
I don't think we need these loggers since they will just panic if they fail anyways